js/lib/ui.js

```diff
@@ -368,8 +368,10 @@ elgg.ui.initDatePicker = function() {
  * name is remove_friend, you should call this function with "remove-friend" instead.
  */
 elgg.ui.registerTogglableMenuItems = function(menuItemNameA, menuItemNameB) {
+
 	// Handles clicking the first button.
-	$(document).on('click', '.elgg-menu-item-' + menuItemNameA + ' a', function() {
+	$(document).off('click.togglable', '.elgg-menu-item-' + menuItemNameA + ' a')
```

Comment:
It's a namespaced event, which is triggered whenever `click` is triggered. You could also trigger it exclusively by `trigger('click.toggleable')`, or with all events in the namespace with `trigger('.toggleable')`.

Which makes me think perhaps using `on('elgg.ui.toggle')` in your other PR is not the safest pattern...